cmd/crictl/container.go

```diff
@@ -351,7 +351,12 @@ var removeContainerCommand = &cli.Command{
 		}
 
 		if len(ids) == 0 {
-			return cli.ShowSubcommandHelp(ctx)
+			if ctx.Bool("all") {
+				logrus.Info("No containers to remove")
+				return nil
+			} else {
+				return cli.ShowSubcommandHelp(ctx)
```

Comment:
maybe add a test case to https://github.com/kubernetes-sigs/cri-tools/blob/master/test/e2e/help_test.go

Reply:
done but not tested (let see if the CI is happy)

Reply:
yeah, likely need to start crio or something before calling in